virttest/utils_misc.py

```diff
@@ -1088,7 +1088,8 @@ def umount(src, mount_point, fstype, verbose=False, fstype_mtab=None):
         return True
 
 
-def mount(src, mount_point, fstype, perm=None, verbose=False, fstype_mtab=None):
+def mount(src, mount_point, fstype, perm=None, verbose=False, fstype_mtab=None,
+          session=None):
```

Comment:
Also can we consider support for umount as it will be used in cleanup code, otherwise code looks good to me.